x/ibc/20-transfer/keeper/relay.go

```diff
@@ -203,6 +211,12 @@ func (k Keeper) refundPacketAmount(ctx sdk.Context, packet channel.Packet, data
 	prefix := types.GetDenomPrefix(packet.GetSourcePort(), packet.GetSourceChannel())
 	source := strings.HasPrefix(data.Amount[0].Denom, prefix)
 
+	// decode the sender address
+	sender, err := sdk.AccAddressFromBech32(data.Sender)
```

Comment:
ditto

Reply:
See above comment. We sent this packet, so we encoded the sender address with our bech32 prefix.